BookingProject/BookingWebAPI/BookingWebApi.Tests/Controllers/AccountControllerTest.cs

```diff
@@ -0,0 +1,112 @@
+﻿using BookingWebApi.Application.DTOs.AppUserDTOs;
+using BookingWebApi.Application.Interfaces;
+using BookingWebApi.Application.Models;
+using BookingWebApi.Controllers;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookingWebApi.Tests.Controllers
+{
+    public class AccountControllerTest
+    {
+        private readonly Mock<IAuthenticationService> _authService;
+        private readonly AccountController _controller;
+
+        public AccountControllerTest()
+        {
+            _authService = new Mock<IAuthenticationService>();
+            _controller = new AccountController(_authService.Object);
```

Comment:
We don't unit test controllers. Controllers are used in integration testing. Instead, unit test you Application layer services